h/templates/client/annotation.html

```diff
@@ -71,7 +71,7 @@
   <!-- Body -->
   <section name="text" class="annotation-body">
     <excerpt enabled="vm.feature('truncate_annotations') && !vm.editing()">
-      <markdown ng-model="vm.annotation.text"
+      <markdown ng-model="vm.form.text"
```

Comment:
Definitely helpful to have this distinction between the properties of an annotation that are editable in the card and those that are not.